sormas-backend/src/test/java/de/symeda/sormas/backend/person/PersonFacadeEjbTest.java

```diff
@@ -1778,4 +1778,128 @@ public void testIsEditAllowedLinkedToNotOwnedCaseAndDeletedAssociatedObject() {
 		getImmunizationFacade().delete(immunization.getUuid(), new DeletionDetails());
 		assertFalse(getPersonFacade().isEditAllowed(person.getUuid()));
 	}
+
+	@Test
+	public void testIsSharedOrReceived() {
+		RDCF rdcf = creator.createRDCF();
+		UserDto userDto = creator.createUser(rdcf, creator.getUserRoleReference(DefaultUserRole.NATIONAL_USER));
+		User user = getUserService().getByUuid(userDto.getUuid());
+
+		PersonDto person = creator.createPerson();
+		CaseDataDto sharedCase = creator.createCase(userDto.toReference(), person.toReference(), rdcf, null);
+
+		assertFalse(getPersonFacade().isSharedOrReceived(person.getUuid()));
+
+		// shared case
+		creator.createShareRequestInfo(
```

Comment:
general remark: I think it would be beneficial to crate a `creator.createSharedCase` etc method by now? It is frequently encountered.

Reply:
can't use creator.createSharedCase because the re is an assert between creating the "case and sharing" it